x-pack/plugins/security_solution/public/flyout/document_details/right/components/alert_status.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiTitle } from '@elastic/eui';
+import { getUserDisplayName } from '@kbn/user-profile-components';
+import { FormattedMessage } from '@kbn/i18n-react';
+import type { FC } from 'react';
+import React from 'react';
+import { WORKFLOW_STATUS_DETAILS_TEST_ID, WORKFLOW_STATUS_TITLE_TEST_ID } from './test_ids';
+import { useRightPanelContext } from '../context';
+import { useBulkGetUserProfiles } from '../../../../common/components/user_profiles/use_bulk_get_user_profiles';
+import { PreferenceFormattedDate } from '../../../../common/components/formatted_date';
+
+export const AlertStatus: FC = () => {
+  const { searchHit } = useRightPanelContext();
+  const statusUpdatedBy = searchHit.fields?.['kibana.alert.workflow_user'] as string;
+  const statusUpdatedAt = searchHit.fields?.['kibana.alert.workflow_status_updated_at'];
```

Comment:
any reason for not using [getFieldsData](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/flyout/document_details/right/context.tsx#L62) along side our x-pack/plugins/security_solution/public/flyout/document_details/shared/utils.tsx like we use in the rest of the document_details flyout code?

The `getFieldsData` is available from the `useRightPanelContext` you're already using.

It would be
```
const { getFieldsData } = useRightPanelContext();

const statusUpdatedBy = getField(getFieldsData('kibana.alert.workflow_user'));
const statusUpdatedAt = getField(getFieldsData('kibana.alert.workflow_status_updated_at'));
```

Reply:
I figured getting the raw data from the searchHit was simple enough, I can change it though